ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -8434,6 +8502,13 @@ FPHC:
   - heating_water_valve_percentage_command
   - chilled_water_valve_percentage_command
 
+SVVM:
+  guid: "9b702772-2303-4648-bc88-4f23d97c0349"
+  description: "Steam valve voltage monitoring."
+  is_abstract: true
+  uses:
+  - steam_valve_voltage_command
```

Comment:
is there a sensor? does this control to any air temperature?

Reply:
No way to confirm if this controls air temperature. Also steam_valve_voltage was seen on HX & HWS besides AHUs

Reply:
In general it is preferable for there to be feedback anythere we have a command. I would recommend adding a steam_valve_voltage_sensor, changing the type to contro;, and marking it missing in the building config.

Reply:
@shambergoldstein Added voltage sensor as a required field for SVVM